server/data/assessForEarlyReleaseApiClient.ts

```diff
@@ -1,3 +1,4 @@
+import { OffenderSummary } from '../@types/assessForEarlyReleaseApiClientTypes'
```

Comment:
**nit:** super trivial but might be good to get into the habit of `import type { OffenderSummary}` or `import { type OffenderSummary }` (when importing some types  and other non types). 

Then it removes the possibility of triggering runtime side effects of importing a module